app/models/ability.rb

```diff
@@ -66,8 +66,7 @@ def initialize(user)
     end
 
     can :export, Group do |group|
-      user.is_admin? or
```

Comment:
I understand user.is_admin = true means user has access to loomio.org/admin, eh?